src/main/java/ch/njol/skript/expressions/ExprActiveItem.java

```diff
@@ -0,0 +1,79 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Active Item")
+@Description(
+		"Returns the item the entity is currently using (ie: the food they're eating, " +
+		"the bow they're drawing back, etc.). This cannot be changed. " +
+		"If an entity is not using any item, this will return 0 air."
+)
+@Examples({
+		"on damage of player:",
+		"\tif victim's active tool is a bow:",
+		"\t\tinterrupt player's active item use"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Paper")
+public class ExprActiveItem extends PropertyExpression<LivingEntity, ItemType> {
```

Comment:
Use ItemStack not ItemType. ItemType is to only be used in the context of a literal.

Reply:
Tud's fault on the ItemType change. It should be an ItemStack.

@UnderscoreTud's question about this being a SimplePropertyExpression was not answered or committed. I also had a change about that.

Reply:
Ah I missed that with tud, yeah it can definitely be a SPE.